capa/main.py

```diff
@@ -778,7 +786,7 @@ def main(argv=None):
         logger.debug("using rules path: %s", rules_path)
 
     try:
-        rules = get_rules(rules_path, disable_progress=args.quiet)
+        rules = get_rules(rules_path, disable_progress=args.quiet or args.tag)
```

Comment:
can you explain this change?

Reply:
this is a workaround to show that we only load a limited number of rules (the change below)
here we omit the output if a tag is given

Reply:
why can we not show the output if a tag is given? 🤔 I mean we still load all the rules and filter afterwards. Are the rules really skipped? a wrong formatted rules which is not included in the tag would still make capa fail, right? I find the output a bit confusing.

This also make me thing, that maybe we shouldn't load all the rules in this case. We could filter by the rule directory. It may speed capa and ignore possible problems with rules that aren't being used. 🤔 Or am I missing something? 🤔

Reply:
I've removed this update. Let's leave this unchanged.